src/app/modules/shared-components/components/score-ring/score-ring.component.ts

```diff
@@ -22,6 +23,7 @@ export class ScoreRingComponent implements OnChanges {
   @Input() icon?: string; // a font-awesome icon identifier
   @Input() scoreFillColor?: string;
   @Input() forTree = false;
+  @Input() isDarkBg = false;
```

Comment:
I think the name `isDarkBg` is misleading by indicating a dark background inside the score-ring component, like `isDark` changes the _inner_ text color
I think something like `forDarkBg` would be more suited, not sure though.
What do you think ?

Reply:
Perfect, fixed